portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,21 @@
+body{
```

Comment:
Use an indentation of 2 spaces for CSS and add a space between a selector and the opening `{`, as you can see in our style guide examples:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Thanks, I'll go ahead and push through a commit with the required spacing and a separate commit for the Indentation.